Formula/groovy-mode.rb

```diff
@@ -0,0 +1,55 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class GroovyMode < EmacsFormula
+  desc "Modes for Groovy and Groovy-related technology"
+  homepage "https://github.com/Groovy-Emacs-Modes/groovy-emacs-modes"
+  head "https://github.com/Groovy-Emacs-Modes/groovy-emacs-modes.git"
+
+  depends_on :emacs => "22.1"
+
+  def install
+    el_array = %w[groovy-mode.el groovy-electric.el inf-groovy.el]
+    byte_compile el_array
+    elisp.install el_array, Dir["*.elc"]
+
+    doc.install "README.md", "gplv2.txt"
+  end
+
+  def caveats; <<-EOS.undent
```

Comment:
I've started removing caveats that aren't Homebrew-specific information, since they can easily get out-of-date without us noticing.  Can you shorten this just to the part about grails-mode?